src/components/DefineCompany/YourCompany.vue

```diff
@@ -0,0 +1,247 @@
+<template>
+  <v-card flat>
+     <div class="define-company-header">
+        <v-icon>mdi-domain</v-icon>
+        <label class="define-company-title"><strong>Your Company</strong></label>
+    </div>
+
+    <div class="section-container px-8">
+      <!--TODO: Replace container content with Name Request Summary when it is ready -->
```

Comment:
@cameron-freshworks This is where your thing will fit it, maybe.

Reply:
For sure, very similar to what i did during testing, but the section component actually slotted in 'after' the `Company Name` label.
 We obviously haven't see the designs from Tracey but as per the wireframes, the initial column ( md4 ) might be a little too spacious going forward. I found that md3 followed by an md9 was a little closer to the designs but we can assess that going forward because it would involve changing all the columns everywhere for alignment. Just something to think about :)

Reply:
Yes, your component will go "somewhere" around here :)
We may have to rework the columns because they don't seem to keep their preferred widths between rows. But let's handle that when we rework the rest of the components.

Reply:
Absolutely!